pkg/release/annotator.go

```diff
@@ -14,7 +14,7 @@ import (
 
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
-	"github.com/fluxcd/helm-operator/pkg/apis/helm.fluxcd.io/v1"
+	helmfluxcdiov1 "github.com/fluxcd/helm-operator/pkg/apis/helm.fluxcd.io/v1"
```

Comment:
Changed to be consistent with other modules as my linter was adding the v1 again.